gossip/src/crds.rs

```diff
@@ -23,6 +23,7 @@
 //!
 //! A value is updated to a new version if the labels match, and the value
 //! wallclock is later, or the value hash is greater.
+#![allow(clippy::integer_arithmetic)]
```

Comment:
I don't think we need this. It is already at the crate level:
https://github.com/solana-labs/solana/blob/c69bc00f6/gossip/src/lib.rs#L2